deps/wazuh_testing/wazuh_testing/tools/__init__.py

```diff
@@ -45,18 +46,31 @@
     except (ImportError, KeyError, ModuleNotFoundError):
         pass
 
-if sys.platform == 'darwin' or sys.platform == 'win32' or sys.platform == 'sunos5':
-    WAZUH_SERVICE = 'wazuh.agent'
-else:
-    try:
-        with open(os.path.join(WAZUH_PATH, 'etc/ossec-init.conf'), 'r') as f:
-            type_ = None
-            for line in f.readlines():
-                if 'TYPE' in line:
-                    type_ = line.split('"')[1]
-            WAZUH_SERVICE = 'wazuh-manager' if type_ == 'server' else 'wazuh-agent'
-    except FileNotFoundError:
-        pass
+
+def get_version():
+
+    if platform.system() in ['Windows', 'win32']:
+        with open(os.path.join(WAZUH_PATH, 'VERSION'), 'r') as f:
+            version = f.read()
+            return version[:version.rfind('\n')]
+
+    else:  # Linux, sunos5, darwin, aix...
+        return subprocess.check_output([
+          f"{WAZUH_PATH}/bin/wazuh-control", "info", "-v"
+        ], stderr=subprocess.PIPE).decode('utf-8')
+
+
+def get_service():
+    if platform.system() in ['Windows', 'win32']:
+        return 'wazuh-agent'
+
+    else:  # Linux, sunos5, darwin, aix...
+        service = subprocess.check_output([
+          f"{WAZUH_PATH}/bin/wazuh-control", "info", "-t"
+        ], stderr=subprocess.PIPE).decode('utf-8')
+
+    return 'wazuh-manager' if service == 'server' else 'wazuh-agent'
```

Comment:
![Screenshot from 2021-02-05 01-02-59](https://user-images.githubusercontent.com/11434230/106988221-f8d84d80-674d-11eb-87d0-3983ab35cfec.png)
During a test the following problem was identified:
**wazuh-control** responds with a trailing **\n**. which makes the comparison fails and returns **wazuh-agent** for manager installations.

Reply:
Thank you! This should fix it: https://github.com/wazuh/wazuh-qa/pull/1013/commits/4f69e1174d279fb5cff42a013853b95216f3d9ca